src/styles/_header.scss

```diff
@@ -0,0 +1,82 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__link {
+  color: $main-color;
+  text-decoration: none;
+  text-transform: uppercase;
+  display: inline-flex;
+  height: 60px;
+  align-items: center;
+  justify-content: center;
+  line-height: 14.06px;
+  transition: all 0.3s;
+
+  &:hover {
+    color: $blue-accent;
+  }
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
+  text-decoration: none;
+  list-style-type: none;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.link:hover {
```

Comment:
The selector '.link:hover' is too generic and might conflict with other styles. Consider using a more specific class name to avoid potential conflicts.